recipes/console_bridge/LICENSE

```diff
@@ -0,0 +1,33 @@
+/*********************************************************************
```

Comment:
I am bundling the LICENSE file here since there is no license file in the console_bridge repo. I opened a PR upstream: https://github.com/ros/console_bridge/pull/62.